Makefile

```diff
@@ -53,3 +54,6 @@ install:
 uninstall:
 	/bin/rm -f $(DESTDIR)/$(MODULE_DESTDIR)/nv_peer_mem.ko
 	if [ ! -n "$(DESTDIR)" ]; then $(DEPMOD) -r -ae $(KVER);fi;
+
+gen_nv_symvers:
+	$(PWD)/create_nv.symvers.sh $(KVER)
```

Comment:
hmm, looking at the rules file, I see that it just defines it and never use it... there are also other lines that do not make sense and not needed. I will clean them up in a separate patch as well.

for for DKMS package, this patch already copies create_nv.symvers.sh instread of nv.symvers into the DKMS sources/package.